src/themes/components/tabs.ts

```diff
@@ -0,0 +1,41 @@
+const baseStyleTab = {
+  textStyle: 'body',
+  transitionProperty: 'common',
+  transitionDuration: 'normal',
+  borderBottom: '1.5px solid',
+  borderColor: 'transparent',
+  p: 'sm',
+  marginBottom: '-1.5px',
+  color: 'gray.600',
+  _selected: {
+    fontWeight: 'bold',
+    color: 'gray.900',
+    borderColor: 'currentColor',
+  },
+  _disabled: {
+    _active: { bg: 'none' },
+    cursor: 'not-allowed',
+    opacity: 0.4,
+  },
+};
+
+const baseStyleTablist = {
+  borderBottom: '1.5px solid',
```

Comment:
We don't have this token in place. Is it possible to check with the designer if we can use 1px or 2px? or if it makes sense for future uses introduce this new token.

Would you mind checking this with the designer? Thank you!